codejail_service/settings/test.py

```diff
@@ -2,15 +2,10 @@
 
 from codejail_service.settings.base import *  # pylint: disable=wildcard-import
 
-# IN-MEMORY TEST DATABASE
+# Django *really* wants a database. So we give it a blank in-memory one.
```

Comment:
Dang. Really? I wonder if we could rig it so it blows up if any unit test actually tries to use it?

Reply:
I'll go ahead and move it to base settings for consistency.